std-bits/table/src/main/java/org/enso/table/operations/RowVisitor.java

```diff
@@ -0,0 +1,9 @@
+package org.enso.table.operations;
+
+public interface RowVisitor {
+
+  void visit(int row);
+
+  default void finalise() {}
+  ;
+}
```

Comment:
nitpick - the name is okay but perhaps it could be a `RowGroupVisitor`?

IMHO the `RowGroupVisitor` makes it clearer that an instance visits a whole group (by calling visit multiple times - per each row - and then finalising).